.goreleaser.yml

```diff
@@ -1,6 +1,8 @@
 env:
   - GO111MODULE=on
   - CGO_ENABLED=0
+  - GITHUB_ORG=knadh
```

Comment:
This allows people to fork and change to their own username and hence test build and publish